.github/workflows/temp-java-ci-upgrade.yaml

```diff
@@ -52,3 +52,8 @@ jobs:
             :sda-commons-starter-s3:test
             :sda-commons-starter-web:test
             :sda-commons-web-testing:test
+            :sda-commons-cloudevents:test
```

Comment:
Seems like the regular build (Windows still running) is green now. Should we remove this temporary workflow and rely on the regular checks?